### ISSUE TITLE
How i can config load_balancing_config.mode with terraform?

### ISSUE THREAD

**Issue by evsasha** [yandex-cloud/terraform-provider-yandex#262]
Hello, I don't see any way to choose [balancing mode](https://cloud.yandex.ru/docs/application-load-balancer/concepts/backend-group#balancing-mode) for ALB via terrafrom in [yandex_alb_backend_group ](https://registry.terraform.io/providers/yandex-cloud/yandex/latest/docs/resources/alb_backend_group) resource.

In the web console, I have option to set the balancing mode for the backend, and this can be done [through the API](https://cloud.yandex.ru/docs/application-load-balancer/api-ref/BackendGroup/addBackend). But it is impossible to do this through the terraform provider.

**Comment by evgenymarkov** [comment]
Faced with the same problem. It is not possible to set load_balancing_mode using Terraform. There is also no documentation about session_affinity.

**Comment by evsasha** [comment]
Right now I'm using this workaround:
* Initiate YC CLI with service account credentials (authorization key.json file)
* Get an IAM token
* Make an API request to update the backend

This is very painful..

```hcl
# Save key.json auth file for YC CLI
resource "local_sensitive_file" "key_json" {
    content  = "${var.key_json}"
    filename = "${path.module}/key.json"
}

# Temporary file with IAM token
resource "local_sensitive_file" "token_txt" {
    content  = "placeholder"
    filename = "${path.module}/token.txt"
}

resource "null_resource" "alb_bg_maglev_hash" {

  # Resource depend on ALB main resource
  depends_on = [
    yandex_alb_load_balancer.alb
  ]

  # Recreate resource on each run
  triggers = {
    always_run = "${timestamp()}"
  }

  # Create YC profile for service account, skip error if profile already exist
  provisioner "local-exec" {
    command = "yc config profile create sa-profile || true"
  }

  # Activate YC profile
  provisioner "local-exec" {
    command = "yc config profile activate sa-profile"
  }
  
  # Get auth info from file
  provisioner "local-exec" {
    command = "yc config set service-account-key ${local_sensitive_file.key_json.filename}"
  }

  # Get IAM token and save it to temp file
  provisioner "local-exec" {
    command = "yc iam create-token | tr -d '\n' > ${local_sensitive_file.token_txt.filename}"
  }

  # Update loadBalancingConfig.mode for backend
  provisioner "local-exec" {
    command = <<EOT
        curl -s https://alb.api.cloud.yandex.net/apploadbalancer/v1/backendGroups/${yandex_alb_backend_group.example_bg.id}:updateBackend -H 'Authorization: Bearer ${file(local_sensitive_file.token_txt.filename)}' -d '{"updateMask":"http.loadBalancingConfig","http":{"name": "${yandex_alb_backend_group.example_bg.http_backend[0].name}","loadBalancingConfig":{"mode":"MAGLEV_HASH"}}}'
    EOT
  }

  # Delete temp files
  provisioner "local-exec" {
    command = "rm -rf ${local_sensitive_file.token_txt.filename} ${local_sensitive_file.key_json.filename}"
  }
}

```

**Comment by evsasha** [comment]
Fixed in version `0.76`.

**Comment by evsasha** [comment]
Fixed in version 0.76.
